docs/meshstack.gcp.index.md

```diff
@@ -234,3 +234,28 @@ We recommend that configuration include at least 3 characters of the random para
       "%.15s-%.10s-%.3s"
 }
 ```
+
+## Audit Logs for meshfed-service User 
+
+The actions of the `meshfed-service` User can be monitored via [Audit Logs](https://cloud.google.com/logging/docs/audit/). This allows an in-depth view meshStack activities for GCP project at any moment.
+
+### Enable Audit Logs
+
+> Enabling Audit Logs may incur charges.
+
+meshcloud recommends to enable Audit Logs on the organizational level for monitoring `meshfed-service` User. This is achivied by following these steps:
+
+1. Navigate to the organizational level in [GCP Cloud Console](https://console.cloud.google.com/)
+2. Navigate to [IAM & Admin --> Audit logs](https://console.cloud.google.com/iam-admin/audit)
+3. Filter the table for `Cloud Resource Manager API` and select the resulting entry
```

Comment:
d: https://docs.dev.meshcloud.io/docs/meshstack.gcp.index.html#meshfed-service-iam-role documents that we also have further rights that we may use (billing, serviceusage, deploymentmanager). 

I think we shouldmention that there is be a correspondence between the rights granted to the service user and the namespaces operators should enable for auditing.